### PR TITLE
add lighthouse-user-flows doc

### LINKED ISSUE #6527: [Vitals] Lighthouse User Flows [11/3/2021]
* Proposed title: Lighthouse User Flows
* Proposed by: bckenny
* Related areas: Web Vitals/Performance
* Content type: How-To/API Guide (https://documentation.divio.com/how-to-guides/)
* Deadline: 11/3/2021
* Deadline explanation: Mentioned in the keynote, and we'd like to start getting feedback from users trying out the new API.
* [Draft](https://docs.google.com/document/d/1fCBn4xL5dvlF5CX8uQFEFj631GU34Q2pkjjZ_33CS8I/edit?usp=sharing&resourcekey=0-nBp7yMpHX9KiaTObmu1mXA)
* Content summary: Lighthouse has previously been limited to auditing initial page loads. This article explains Lighthouse's new user-flow support, which allows invoking Lighthouse at any time within a Puppeteer-scripted interaction with a site. Several examples (with source code) are given for testing scenarios that weren't possible in Lighthouse before.

### PR DESCRIPTION
**Pull request by brendankenny**
Fixes #6527

### CODE REVIEW

**Comment by malchata** [comment]
Reviewing now!

File: src/site/content/en/blog/lighthouse-user-flows/index.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
+---
+title: 'Lighthouse user flows'
+subhead: Try out a new Lighthouse API to measure throughout a user flow.
+description: Try out a new Lighthouse API to measure throughout a user flow.
```
Comment (malchata) [inline src/site/content/en/blog/lighthouse-user-flows/index.md:4]: Is there a word missing in these? "To measure [potentially missed word] through a user flow?"

File: src/site/content/en/blog/lighthouse-user-flows/index.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+hero: image/MtjnObpuceYe3ijODN3a79WrxLU2/NbfbZ81Sx2ZlFvRwTiBV.png
+alt: A Lighthouse flow report summary
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - performance
+  - web-vitals
+  - lighthouse
+---
+
+Lighthouse is a fantastic tool for testing performance and best practices during initial page load. However, it's traditionally been difficult to use Lighthouse to analyze other aspects of the life of a page. Page loads with a warm cache, or an activated Service Worker, or during user interactions are all scenarios that have been difficult to test using Lighthouse.
```
Comment (malchata) [inline src/site/content/en/blog/lighthouse-user-flows/index.md:17]:
```suggestion
Lighthouse is a fantastic tool for testing performance and best practices during initial page load. However, it's traditionally been difficult to use Lighthouse to analyze other aspects of the life of a page, such as:

- Page loads with a warm cache
- Pages with an activated Service Worker
- Accounting for potential user interactions
```

**Review by malchata — APPROVED** [review]
All suggestions are pretty much optional, so apply them at your discretion. Otherwise, LGTM!

File: src/site/content/en/blog/lighthouse-user-flows/index.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+  const report = flow.generateReport();
+  fs.writeFileSync('flow.report.html', report);
+  open('flow.report.html', {wait: false});
+}
+
+captureReport();
+```
+
+This is the simplest "flow". When opened, the report shows a summary view with only the single step; clicking on that step will reveal a traditional Lighthouse report for that navigation.
```
[the remark this answers is not in the document]
Comment (brendankenny) [reply src/site/content/en/blog/lighthouse-user-flows/index.md:79]: > Not sure "flow" needs to be in quotes in this case.

ha, almost seems to be mocking our own name for it :)

**Comment by brendankenny** [comment]
thanks @malchata!

**Comment by philipwalton** [comment]
@googlebot I consent.

File: src/site/content/en/blog/lighthouse-user-flows/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (malchata) [reply src/site/content/en/blog/lighthouse-user-flows/index.md]: I've totally done this, too!

**Pull request merged**